stairway/src/test/java/bio/terra/stairway/impl/EnumerateFlightsTest.java

```diff
@@ -146,14 +148,26 @@ public void enumTest() throws Exception {
     flightEnum = flightDao.getFlights(pageTokenString, 3, filter);
     checkResults("case 10", flightEnum.getFlightStateList(), Arrays.asList("3", "4", "5"));
     assertThat(flightEnum.getTotalFlights(), equalTo(6));
+
+    // Case 11: sorting in ascending order
+    filter =
+            new FlightFilter()
+                    .submittedTimeSortDirection(FlightFilterSortDirection.ASC);
+    flightList = flightDao.getFlights(0, 100, filter);
+    checkResults("case 11", flightList, Arrays.asList("0", "1", "2", "3", "4", "5"));
```

Comment:
You could avoid duplicating the list of flight IDs by using `flights`.
```suggestion
    checkResults("case 11", flightList, flights.stream().map(FlightState::getFlightId).collect(Collectors.toList()));
```

Reply:
Yeah, but I prefer the visibility of seeing what we're testing here